cmd/hubagent/main.go

```diff
@@ -93,13 +94,25 @@ func main() {
 	}
 
 	klog.V(2).InfoS("starting hubagent")
-
-	if err = (&mcv1alpha1.Reconciler{
-		Client:                  mgr.GetClient(),
-		NetworkingAgentsEnabled: opts.NetworkingAgentsEnabled,
-	}).SetupWithManager(mgr); err != nil {
-		klog.ErrorS(err, "unable to create v1alpha1 controller", "controller", "MemberCluster")
-		exitWithErrorFunc()
+	if opts.EnableV1Alpha1APIs {
```

Comment:
Just curiois, this is intentionally made to be not mutually exclusive I assume?

Reply:
'cause the code in other files from this PR is modified to support both controllers.